### ISSUE TITLE
Lovelance Interface 

### ISSUE THREAD

**Issue by mrochette01** [home-assistant/frontend#10562]
### The problem

To select "Configuration" and "Supervisor", I have to point the mouse on the icon only. The tab name area is not active anymore.

### What version of Home Assistant Core has the issue?

	core-2021.11.1

### What was the last working version of Home Assistant Core?

code-2021-10.7

### What type of installation are you running?

Home Assistant OS

### Integration causing the issue

Lovelance

**Comment by spacegaier** [comment]
Can you please provide some more details (e.g. what "tab" are you talking about) and perhaps a screenshot. I am either not getting your issue or (if I correctly interpreted your short description correctly) cannot reproduce it.

**Comment by mrochette01** [comment]
Hi, I tried to reproduce the situation I had experienced many times but without success. For your information, I use the Chrome browser (latest version) as well as Windows 11. I suspect that the interface problem was due to Windows 11 or Chrome. I have noticed the occasional disappearing icons in the taskbar at the bottom of the screen. I suggest closing my post. I will still keep watching though. Previously I was an X10 user for 20 years, later Zwave (Vera controller) (10 years) and now Home Assistant (Zwave and Zigbee). I am very happy with it.
